### PR TITLE
[HOPSWORKS-2382] Collect services logs in Elastic

### PR DESCRIPTION
**Pull request by SirOibaf**

### CODE REVIEW

File: providers/start.rb [hunk 1]
```diff
@@ -223,80 +223,35 @@
...
                 "type":"keyword"
              },
-             "thread":{
+             "host":{
                 "type":"keyword"
              },
-             "file":{
-                "type":"keyword"
+             "priority":{
+                "type":"text"
```
Comment (o-alex) [inline providers/start.rb:245]: are you sure you need to create priority as text and KEYWORD is not enough? are you going to run partial matches on this? seems to me like you will always search by enum like constructs: DEBUG/INFO

File: templates/default/roles.yml.erb [hunk 2]
```diff
@@ -45,8 +45,7 @@ logstash_hopsworks:
     - index_patterns:
       - "*_logs-*"
       - "*_serving-*"
-      - "*_beamjobserver-*"
-      - "*_beamsdkworker-*"
+      - ".services-*"
```
Comment (o-alex) [inline templates/default/roles.yml.erb:48]: As I pointed in the pm on slack, this roles index pattern is at odds with the one in the elastic pattern definition where you set it as:
 `"index_patterns":[
    "*.services-*"
 ]`

File: templates/default/roles.yml.erb [hunk 3]
```diff
@@ -137,6 +136,25 @@ elastic_exporter_hopsworks:
     - cluster:monitor/nodes/info
     - indices:data/read/scroll/clear
 
+service_log_viewer:
+  description: "Provide the minimum permissions required to grant readonly access for Hopsworks admins to service logs"
+  reserved: true
+  index_permissions:
+    - index_patterns:
+      - ".services-*"
```
Comment (o-alex) [inline templates/default/roles.yml.erb:144]: same as above with the pattern issue

**Review by o-alex — CHANGES_REQUESTED** [review]

File: templates/default/roles.yml.erb (hunk 2 above)
Comment (SirOibaf) [reply templates/default/roles.yml.erb:48]: `.services-*` is the correct one. One index for all the logs.

File: templates/default/roles.yml.erb (hunk 3 above)
Comment (SirOibaf) [reply templates/default/roles.yml.erb:144]: Same as above.

File: providers/start.rb (hunk 1 above)
Comment (SirOibaf) [reply providers/start.rb:245]: Done.

**Review by o-alex — APPROVED** [review]

**Review by kouzant — APPROVED** [review]

**Pull request merged**